### ISSUE TITLE
[Premium] How to compare baseline prices of EP1 plan

### ISSUE THREAD

**Issue by ishepherd** [Azure/Azure-Functions#1445]
The only way I found to get a price is: pick a region, select in tab 1 of "New Function App" wizard, go to tab 2, look at "Spec Picker". Repeat for the next region, and the next, ...

This method showed me that East US 2 is 3% cheaper than West US 2.

Is East US 2 the cheapest or are there cheaper?

Can you publish a list to compare easily, like https://azure.microsoft.com/en-us/pricing/details/app-service/windows/ is published for normal app service plans?

----

FYI there seems to be trouble in North Central US. The spec picker finds no price there. On my first few attempts of plan creation, I got [this error written up by @jeffhollan](https://github.com/Azure/Azure-Functions/wiki/Creating-Function-Apps-in-an-existing-Resource-Group) - even though it was a _new_ resource group.

**Comment by ishepherd** [comment]
TL;DR please add "ElasticPremium" SKUs to https://azure.microsoft.com/en-us/pricing/details/app-service/windows/ - Or tell me if they are already published somewhere else. Thanks

**Comment by alexkarcher-msft** [comment]
This is the page with Functions Premium pricing, and there is a region picker at the top 
https://azure.microsoft.com/en-us/pricing/details/functions/

You do need to multiply by the GB and CPU count for your desired SKU here:  https://docs.microsoft.com/en-us/azure/azure-functions/functions-premium-plan#available-instance-skus

US East 2 and US West 2 are the least expensive regions.
If you're feeling charitable, France South is the most expensive region 😄

**Comment by ishepherd** [comment]
Thanks Alex

Source                 | Actual WUS2 | Actual EUS2
-------------------- | ----------- | -----------
| [Premium pricing page](https://azure.microsoft.com/en-us/pricing/details/functions/) (1 vCPU & 3.5 GB) | $145.927 + consumption | $145.927 + consumption |
| Spec picker estimate |  $74.03 baseline | $71.69 baseline |
| Azure Cost Analysis data | (~2.50 per day) => ~$76  | not tried |

I am curious why East US 2 is estimated as less than West US 2. Oh well, it's not so different.

I need to know: has GA pricing kicked in yet?
* the pricing page says
> GA pricing effective January 1, 2020
* but the price shown in Cost Analysis is still (Jan 6) only ~half what the pricing page quotes

**Comment by alexkarcher-msft** [comment]
Hello, 

The pricing page is the ultimate source of truth for our billing rate. 

We are aware that the spec picker is still estimating preview prices and are updating it in this sprint to show correct baselines.

As for cost analysis, I'm not sure where they get their info, but it looks like it is just a historical record of what you have paid over the last billing cycle. I would expect that over the next 30 days your estimate would creep closer and closer to the GA rate.

**Comment by ishepherd** [comment]
Thanks @alexkarcher-msft, but this was the wrong answer. Our premium plan app already looked expensive to justify and now you have to go and double it 😭 

Thanks for your help though 🤣

**Comment by onionhammer** [comment]
EP1 plans seem to run about 2x higher cost than advertised; which is at 157/mo (~ $5/day) for EP1 in Central US, but about $9-$10/day in practice.. even with only a couple thousand invocations per day, and since the metrics lack any information about the scale it's impossible to tell which functions are causing the scale to spike up, or even if things like Cosmos changefeed triggers vs service bus triggers are causing scale-outs.

**Comment by petro2050** [comment]
Premium instances are too expensive, IMO. The performance and reliability of consumption instances is subpar, so we're forced to shell out for the privilege of running a dedicated function instance.
